doc/tools/build.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# coding: utf-8
+
+# Builds GD Script API
+# Requires Python 3.4+
+# Run with --help or see print_usage 
+
+import sys
+if(sys.version_info<(3,4)):
+    print("Please upgrade python to version 3.4 or higher.")
+    print("Your version: %s\n" %sys.version_info)
+    sys.exit(1)
+
+from pathlib import Path
+
+# Configure these variables for your system or specify on the command line
+
+GD=str(Path.home()) + "/bin/godot-dev"    # Path to Godot executable
+OUT="../api"	        # Path to the output gdscript files  
+
+
+import os
+import glob
+import getopt
+import shutil
+import tempfile 
+import subprocess
+from time import gmtime, strftime
+from xmlproc import process_xml
+
+
+
+def build_gdscript_api(verbose=False):
+
+
+        # Make temp dirs
+        tmp_d = tempfile.mkdtemp()
+        if verbose: print ("Making temp directories in " + tmp_d)
+        os.makedirs(tmp_d + "/doc/classes")
+        os.makedirs(tmp_d + "/modules/voxel/doc_classes")   
+        # Last is not currently used, but will be once VT starts using doc_classes. 
+
+
+        # Make output dir and remove old files
+        if not os.path.isdir(OUT):  
+            if verbose: print ("Making output directory: " + OUT)
+            os.makedirs(OUT)
+
+        for i in glob.glob(OUT + "*.md"):
+            if verbose: print ("Removing old: ", i)
+            os.remove(i)
+
+
+	# Dump XML files from Godot
+        args = [GD, ' --doctool ',  tmp_d ]
+        if verbose:
+            print ("Running: ", args)
+        result = subprocess.run (args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, 
+                universal_newlines=True)
+        if verbose: 
+            print (result.stdout)
+            print("Disregard Godot's errors about files unless they are about Voxel*.")
+
+
+	# Convert files to MD
+        files = glob.glob(tmp_d + "/doc/classes/Voxel*.xml")
+        files += glob.glob(tmp_d + "/modules/voxel/doc_classes/*.xml")
+        count=0
+        for src in files:
+            dest = OUT + Path(src).stem + ".md"
+            if verbose: print("Converting ", src, dest)
+            process_xml(src, dest)
+            count += 1
+
+
+        # Create Class index
+        index=OUT + "Class_List.md"
+        files = sorted(glob.glob(OUT+"Voxel*.md"))
+
+        str  = "# Voxel Tools Class List\n\n" 
+        str += "These classes are exposed to GDScript. This information is also available within the Godot script editor by searching help.\n\n"
+
+        for i in files:
+            f = Path(i).name
+            str += "* [%s](%s)\n" % (f, f)
+
+        str += "\n---\n"
+        str += "* [Doc Index](../01_get-started.md)\n"
+        str += "_Generated on " + strftime("%b %d, %Y", gmtime()) + "_\n" 
+
+        if verbose: print("Writing %s" % index)
```

Comment:
Indent?

Reply:
Indent what? 
_Generated_?

Or indent into the for loop above? No.

Do you not like "if verbose: print" all on one line?

Reply:
It's the inline block yes. I prefer it in two lines, you actually wrote two lines sometimes.